doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Change Data Capture (Postgres)"
+description: "How to create real-time materialized views from PostgreSQL changelog data with Materialize."
+weight: 10
+menu:
+  main:
+    parent: guides
+---
+
+Change Data Capture (CDC) allows you to track and propagate changes in a Postgres database to downstream consumers based on its Write-Ahead Log (WAL). In this guide, we'll cover how to use Materialize to create and efficiently maintain real-time materialized views on top of CDC data.
+
+There are two ways to connect Materialize to a Postgres database for CDC:
+
+* Using the direct [Postgres source](#direct-postgres-source)
+
+* Using [Kafka + Debezium](#kafka--debezium)
+
+## Direct Postgres source
+
+If Kafka is not part of your stack, you can use the [Postgres source](/sql/create-source/postgres/#postgresql-source-details) to connect directly to Materialize. This source uses Postgres’ native replication protocol to continuously ingest changes resulting from `INSERT`, `UPDATE` and `DELETE` operations in the upstream database.
+
+### Setup
+
+**Minimum requirements:** Materialize v0.8.2+, Postgres 10+
+
+Before creating a source in Materialize, you need to ensure that the upstream database is configured to support [logical replication](https://www.postgresql.org/docs/10/logical-replication.html). As a _superuser_:
+
+1. Check the [`wal_level` configuration](https://www.postgresql.org/docs/current/wal-configuration.html) setting:
+
+    ```sql
+    SHOW wal_level;
+    ```
+
+    The default value is `replica`. For CDC, you'll need to set it to `logical` in the database configuration file (`postgresql.conf`). Keep in mind that changing the `wal_level` requires a restart of the Postgres instance and can affect database performance.
+
+    **Note:** Additional steps may be required if you're using Postgres on [Amazon RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.FeatureSupport.LogicalReplication) or [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance).
+
+1. Grant the required privileges to the replication user:
+
+    ```sql
+    ALTER ROLE "user" WITH REPLICATION;
+    ```
+
+    **Note:** This user also needs `SELECT` privileges on the tables you want to replicate, for the initial table sync.
+
+1. Set the replica identity value for the tables you want to replicate:
+
+    ```sql
+    ALTER TABLE repl_table REPLICA IDENTITY FULL;
+    ```
+
+    This setting determines the amount of information that is written to the WAL in case of `UPDATE` and `DELETE` operations. By setting it to `FULL`, change events will include the previous values of all the table’s columns; as a heads-up, you should expect a performance hit in the database from increased CPU usage.
```

Comment:
"Setting it to `FULL` will include the previous values of all the table's columns in change events." And I would make the heads up a different sentence.